src/pages/taesooKim/Main/Feed.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import CommentList from "./CommentList";
+import BtnLike from "./BtnLike";
+import Like from "./Like";
+
+export class Feed extends Component {
+  constructor() {
+    super();
+    this.state = {
+      value: "",
+      comment: [],
+      isClicked: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ comment: this.props.commentList });
+  }
+
+  getValue = e => {
+    this.setState({
+      value: e.target.value,
+    });
+  };
+
+  handleAdd = e => {
+    if (this.state.value) {
+      e.preventDefault();
+      const newComment = {
+        id: Date.now(),
+        userName: "orosy.ts",
+        content: this.state.value,
+        isUser: true,
+      };
+      const comment = [...this.state.comment, newComment];
+      this.setState({ comment, value: "" });
+    }
+    e.preventDefault();
+  };
+
+  handleDelete = id => {
+    const filteredComments = this.state.comment.filter(
+      comment => comment.id !== id
+    );
+    this.setState({ comment: filteredComments });
+  };
+
+  handleClick = () => {
+    this.setState({ isClicked: !this.state.isClicked });
+  };
+
+  render() {
+    const { value, isClicked, comment } = this.state;
+    const { name, profile, description, image, like, update } = this.props;
+    return (
+      <div className="feeds-each">
+        <div className="feeds-profile">
+          <img src={profile} alt="tanggu" />
+          <span>{name}</span>
+          <div className="more-icon">
+            <i className="fas fa-ellipsis-h"></i>
+          </div>
+        </div>
+        <img className="feeds-image" src={image} alt="cookies" />
+        <div className="reaction">
+          <BtnLike isClicked={isClicked} handleClick={this.handleClick} />
+          <div className="plane-icon">
+            <i className="far fa-comment"></i>
+          </div>
+          <div className="plane-icon">
+            <i className="far fa-paper-plane"></i>
+          </div>
+          <div className="bookmark-icon">
+            <i className="far fa-bookmark"></i>
+          </div>
+        </div>
+        <div>
+          {like.map(like => {
```

Comment:
```suggestion
          {likes.map(like => {
```
마찬가지로 복수형 써주시면 좋을거 같아요

Reply:
네네 수정하였습니다🐸